### PR TITLE
Initial project import

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -15,9 +15,88 @@
...
+    help="Perform force copy of documents.",
+    envvar="THOTH_DOCUMENT_SYNC_FORCE",
+)
+@click.argument(
+    "dst",
+    envvar="THOTH_DOCUMENT_SYNC_DST",
+    type=str,
+    metavar="s3://thoth/data/deployment",
+)
+def sync(dst: str, debug: bool = False, force: bool = False) -> None:
```
Comment (pacospace) [inline app.py:53]: what about some metrics also, so we can check if there are any errors about this sync?

Comment (fridex) [reply app.py:53]: Sounds good. Do you want to give this PR a review and open a follow-up for metrics?

Comment (pacospace) [reply app.py:53]: Sure I can do that :)

Comment (pacospace) [reply app.py:53]: is this going to be used in a cronworkflow?

**Review by pacospace — COMMENTED** [review]
lgtm! 💯

**Review by pacospace — APPROVED** [review]

**Comment by fridex** [comment]
> is this going to be used in a cronworkflow?

Yes, it should be a cronworkflow that will make sure data computed in the stage environment are propagated to prod.

**Comment by pacospace** [comment]
> > is this going to be used in a cronworkflow?
> 
> Yes, it should be a cronworkflow that will make sure data computed in the stage environment are propagated to prod.

Perfect thanks!

**Pull request merged**